controllers/quizController.js

```diff
@@ -31,30 +31,51 @@ const addQuiz = async (req, res, next) => {
 const updateQuiz = async (req, res, next) => {
   try {
     const body = { ...req.body };
-    const updatedQuiz = await QuizModel.updateQuizById(req.params.quizId, body);
-    res.status(200).send({
-      payload: updatedQuiz
-    });
+    if(await isOwner(req.params.quizId, req.user._id) || req.user.level=="moderator" || req.user.level=="admin"){
```

Comment:
Need to refactor this. Also use tripple. There should not be an await in If statement